ocs_ci/ocs/resources/pvc.py

```diff
@@ -301,3 +303,32 @@ def get_deviceset_pvs():
     """
     deviceset_pvcs = get_deviceset_pvcs()
     return [pvc.backed_pv_obj for pvc in deviceset_pvcs]
+
+
+def create_pvc_clone(
+        sc_name, parent_pvc, pvc_name=None, do_reload=True
```

Comment:
delete extra indentation

Reply:
On removing the extra line... flake8 gives the following error 
```
ocs_ci/ocs/resources/pvc.py:307:1: E302 expected 2 blank lines, found 1
```

Reply:
Not extra line, extra indentation at 309

Reply:
Yup, got it.